packages/mongodb-memory-server-core/package.json

```diff
@@ -31,6 +31,9 @@
   },
   "homepage": "https://github.com/nodkz/mongodb-memory-server",
   "devDependencies": {
+    "rimraf": "^3.0.0"
+  },
+  "dependencies": {
```

Comment:
like i said in #263, `@type` packages should never be in anything other than `devDependencies`

Reply:
Please see my comments on #263 as Microsoft's TypeScript handbook states you should be including them in dependencies.